src/material/datepicker/month-view.html

```diff
@@ -1,11 +1,17 @@
-<table class="mat-calendar-table" role="presentation">
-  <thead class="mat-calendar-table-header">
-    <tr>
-      <th scope="col" *ngFor="let day of _weekdays" [attr.aria-label]="day.long">{{day.narrow}}</th>
+<table class="mat-calendar-table" role="grid">
+  <thead role="rowgroup" class="mat-calendar-table-header">
+    <tr role="row">
```

Comment:
Aren't the `row` and `columnheader` roles here redundant, given that we're using `tr` and `th` elements?

Reply:
Just to complement: `rowgroup` is also redundant both in `thead` and `tbody` elements.

Reply:
Somehow I got it in my head that one of the screen readers needed the roles to be explicitly set and didn't work with the implicit defaults, but when I tested it again with them removed it worked on VoiceOver, NVDA, and ChromeVox, so that's good enough for me.